projects/rbenv.org/ruby-build/package.yml

```diff
@@ -0,0 +1,36 @@
+distributable:
+  url: git+https://github.com/rbenv/ruby-build.git
+  ref: ${{version.tag}}
+
+versions:
+  github: rbenv/ruby-build
+
+dependencies:
+  gnu.org/autoconf: ^2.72
+  pyyaml.org/libyaml: ^0.2
+  freedesktop.org/pkg-config: ^0.29
+  gnu.org/readline: ^8.2
+  openssl.org: '>=1.1'
+  curl.se: ^8
+  linux:
+    gnu.org/make: '*'
+    gnu.org/gcc: '*'
```

Comment:
I think I need to provide a “build tools” shim package.

Reply:
I am constructing the shims pkg that can replace some of these deps. I'll merge in the mean time.